crates/admin/src/cluster_controller/service.rs

```diff
@@ -320,6 +334,32 @@ impl<T: TransportConnect> Service<T> {
         }
     }
 
+    async fn init_partition_table(&mut self) -> anyhow::Result<()> {
+        let configuration = self.configuration.live_load();
+
+        let partition_table = retry_on_network_error(
```

Comment:
nit: Might be worth logging that we are initializing

Reply:
Yes, will add it.